### PR TITLE
Add github actions to run linting and tests

### PR DESCRIPTION
**Pull request by jkimbo**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def run_tests(self):
     "pytest-benchmark",
     "pytest-cov",
     "pytest-mock",
+    "fastdiff==0.2.0",
```
Comment (wodCZ) [inline setup.py:52]: Why is this pinned? Could use a comment for future reference.

**Comment by wodCZ** [comment]
Comparing with the travis.yml, I think `coveralls` step is missing? https://github.com/graphql-python/graphene/blob/master/.travis.yml#L12-L14

Not experienced with the tooling, but looks good to me other than that 👍

File: setup.py (hunk 1 above)
Comment (jkimbo) [reply setup.py:54]: Comment added.

**Comment by jkimbo** [comment]
@wodCZ I don't think coveralls is that important so I'm going to skip it in favour of getting this merged sooner.

**Pull request merged**